src/files.js

```diff
@@ -70,3 +70,51 @@ export default {
   groceryListWithUserRelations,
   dbIngredients
 }
+
+
+import { 
+  frenchBeans,
+  sprouts,
+  jacketPotato,
+  sweetPotato,
+  kale,
+  leek,
+  broccoli,
+  carrots,
+  mushrooms,
+  onions_shallots,
+  parsnips,
+  peppers,
+  marrow_squash,
+  sweetcorn,
+  courgettes 
+ } from '../cooking_times'
+  
+import cup1 from '../cup1'
+import cup1_2 from '../cup1_2'
+import cup1_4 from '../cup1_4'
+import cup1_8 from '../cup1_8'
+import cup3_4 from '../cup3_4'
+import dash from '../dash'
+import gallon1 from '../gallon1'
+import pinch from '../pinch'
+import pint1 from '../pint1'
+import quart1 from '../quart1'
+import tablespoon_1 from '../tablespoon_1'
+import teaspoon_1 from '../teaspoon_1
+
+export default {
```

Comment:
please add all this variables into block export default that located in up. export default might be one in js and defines in the end